### ISSUE TITLE
Unable to get stable results for multiple iterations (JIGSAW)

### ISSUE THREAD

**Issue by mrizwank97** [nzl-thu/MUSDL#2]
Hi there,

I am using your repository to reproduce the results on the JIGSAW dataset and I am using the default parameter values as defined in the config.py file except for the batch size. I was using the batch size of 1 on colab. But I am unable to get stable results for multiple runs.

For example, I ran USDL for Suturing and got 0.3 test correlation, then I ran it again and this time I got nan value for test correlation, and for the third time, I got .19 test correlation. This behavior is there in all 3 surgical videos. 

What could be the reason for this unstability? What were the hyperparameter values that you have used?

**Comment by nzl-thu** [comment]
Hi, there are 2 main reasons for the unstable results. First, the batch size of 1 is so small, which can lead to serious degradation in Batchnorm(which is used in I3D backbone) performance. Second, the JIGSAWS dataset itself is a very small dataset so the results will not be as stable as datasets like MTL-AQA.
For hyperparameters, we used the same hyperparameters as defined in the config.py.
